src/components/postgresConnect.ts

```diff
@@ -1012,6 +1012,54 @@ class PostgresDB {
     }
   }
 
+  // Permissions policies rely on a entries for user and org in user_org_policy_template table
+  // in the format {userId}.{orgId || 0}.{permissionPolicyId}, templateId
```

Comment:
Just wondering why concatenate userId.orgId.policyId as opposed to just having a separate column for each?